package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "smartsheet",
-  "version": "4.0.1",
+  "version": "4.0.2",
```

Comment:
Can you do another `npm install`? Looks like this version bump wasn't picked up in the `package-lock.json`